src/views/add-project/components/CounterFields/CounterFields.tsx

```diff
@@ -0,0 +1,51 @@
+import { FormControlLabel, Switch } from "@mui/material";
+import { Input } from "components";
+import React, { ChangeEvent } from "react";
+import { AddProjectFormValues } from "../AddProjectForm/AddProjectForm.helpers";
+
+export type CounterFieldsProps = {
+  values: AddProjectFormValues;
+  onChange: (e: ChangeEvent<HTMLInputElement>) => void;
+};
+
+export const CounterFields: React.FC<CounterFieldsProps> = ({
+  values,
```

Comment:
we might be able to destructure here: `{ maxRowCount, currentRowCount, maxRepeatCount, currentRepeatCount }`